test/ResultSet/ReturnType.php

```diff
@@ -0,0 +1,16 @@
+<?php
+namespace ZendTest\Db\ResultSet;
+
+/**
+ * This class is just here to be able to provide the simplest possible custom return type
+ * for a ResultSet when calling ResultSet::setArrayObjectPrototype or using the constructor argument.
+ *
+ * Class ReturnType
```

Comment:
This is quite obvious ;-)

@weierophinney Not sure about the package tag? Does ZF have a standard for this?